test/test_repro-self-gravitating-gas-dynamics.jl

```diff
@@ -1,113 +1,118 @@
+module TestPaperSelfgravitatingGasDynamics
```

Comment:
I think we should rename this file to ` test/test_paper-self-gravitating-gas-dynamics.jl` for consistency.